### PR TITLE
Vote Chart component

### PR DESCRIPTION
**Pull request by joanna-pagepro**
https://tw.auroracreation.com/app/tasks/15536341
image
image

### CODE REVIEW

**Review by TomaszChmielPagepro — APPROVED** [review]

File: src/components/v5/shared/VoteChart/VoteChart.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      </p>
+      <div className="flex pt-2">
+        <div className="flex flex-1 flex-col items-center gap-1">
+          <VoteChartBar
+            value={againstValue}
+            barBackgroundClassName="bg-red-300"
+            direction={VOTE_CHART_BAR_DIRECTION.Left}
+          />
+          <span
+            className={clsx('text-xs text-center transition', {
```
Comment (CzarekDryl) [inline src/components/v5/shared/VoteChart/VoteChart.tsx:46]: Do you need transition here?

File: src/components/v5/shared/VoteChart/VoteChart.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { useIntl } from 'react-intl';
+import { VoteChartProps } from './types';
+import VoteChartBar from './partials/VoteChartBar';
+import { VOTE_CHART_BAR_DIRECTION } from './partials/VoteChartBar/types';
+import VoteChartThresholdIndicator from './partials/VoteChartThresholdIndicator';
+
```
Comment (krawczykkatarzyna) [inline src/components/v5/shared/VoteChart/VoteChart.tsx:8]: displayName is missing

File: src/components/v5/shared/VoteChart/partials/VoteChartBar/VoteChartBar.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { VOTE_CHART_BAR_DIRECTION, VoteChartBarProps } from './types';
+
```
Comment (krawczykkatarzyna) [inline src/components/v5/shared/VoteChart/partials/VoteChartBar/VoteChartBar.tsx:4]: missing  displayName

File: src/components/v5/shared/VoteChart/VoteChart.tsx (hunk 1 above)
Comment (joanna-pagepro) [reply src/components/v5/shared/VoteChart/VoteChart.tsx]: Yes, I want to animate text color change

**Review by CzarekDryl — APPROVED** [review]

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**